integration/.golangci.yml

```diff
@@ -0,0 +1,171 @@
+---
```

Comment:
Why are we putting all of this into an integration/ subdir? Curious the thought process :)

Reply:
Oops, fixed